hawkular-nest/hawkular-nest-itest-parent/hawkular-nest-itest/src/test/java/org/hawkular/nest/StatusEndpointITest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.nest;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.jar.Attributes;
+import java.util.jar.Manifest;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.Response;
+
+/**
+ * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
+ *
+ */
+@RunWith(Arquillian.class)
+public class StatusEndpointITest {
+
+    private static final String statusUrl = "http://127.0.0.1:8080/hawkular/nest/itest/status";
+    private static final String itestWarPath = System.getProperty("hawkular.nest.itest.war.path");
+
+    @Deployment(testable = false)
+    public static WebArchive createDeployment() {
```

Comment:
Hehe, that's a good question. The test war existed before I migrated the tests to Arquillian (I migrated to arquillian to be able to add the c* driver test). The original setup was the test war deployed by copying to the nest deployments directory, which also tested if Nest deploying from there works. So I could not use Arquillian at that point even if I wanted, because I doubt I could persuade Arquillian to deploy via nest/deployments.

Now, that we have the c*driver test that communicates with the embedded c* that was deployed via nest/deployment, the status test war could be migrated to Shrinkwrap. (I am not sure I find it important enough at this point, though.)